processor/src/trace/tests/hasher.rs

```diff
@@ -0,0 +1,163 @@
+use super::{
+    super::{Trace, NUM_RAND_ROWS},
+    build_trace_from_ops_with_inputs, rand_array, Felt, LookupTableRow, Operation, ProgramInputs,
+    Word, ONE, ZERO,
+};
+use crate::hasher::SiblingTableRow;
+use vm_core::{
+    AdviceSet, FieldElement, StarkField, AUX_TRACE_RAND_ELEMENTS, HASHER_AUX_TRACE_OFFSET,
+};
+
+// CONSTANTS
+// ================================================================================================
+
+const P1_COL_IDX: usize = HASHER_AUX_TRACE_OFFSET;
```

Comment:
nit: I think we'll need this index when implementing the hasher aux column AIR constraints, so it might make sense to put it in `core` now so we don't end up with it defined in multiple places.

Reply:
Good point! fixed.